vendor/tfschema/Makefile

```
@@ -0,0 +1,19 @@
+SHELL := /bin/bash
+
+# Package details
+export VENDOR ?= minamijoyo
+export APK_BUILD_TEMPLATE ?= APKBUILD.github-binary
+export PACKAGE_NAME ?= tfschema
+export PACKAGE_REPO_NAME ?= tfschema
+# https://github.com/minamijoyo/tfschema/releases/download/v0.7.0/tfschema_0.7.0_linux_amd64.tar.gz
+export DOWNLOAD_URL ?= $(PACKAGE_REPO_URL)/releases/download/v$(PACKAGE_VERSION)/$(PACKAGE_NAME)_$(PACKAGE_VERSION)_$(OS)_$(ARCH).tar.gz
+export CHECK_COMMAND_ARGUMENTS ?= --version
+export CHECK_COMMAND_ENV ?= HOME=./
```

I don't know where these `CHECK_COMMAND` env vars came from, but they are not used.

Use `APKBUILD_DEPENDS` to support package that needs extra libraries for Alpine
```suggestion
export APKBUILD_DEPENDS += libc6-compat
```